src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --color-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  cursor: pointer;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
```

Comment:
why do we need `display: flex` here?

Reply:
here is a screenshot of the test results with and without this property
![image](https://user-images.githubusercontent.com/116076758/208719613-7ed6a6af-8755-489c-8957-2130c7f795df.png)
![image](https://user-images.githubusercontent.com/116076758/208719802-f8e68e7e-420a-496a-a692-afb669aa68c4.png)